save-core/src/mingwX64Main/kotlin/org/cqfn/save/core/utils/ProcessBuilder.kt

```diff
@@ -0,0 +1,39 @@
+@file:Suppress("FILE_WILDCARD_IMPORTS")
+
+package org.cqfn.save.core.utils
+
+import okio.FileSystem
+import okio.Path
+import platform.posix.*
+
+import kotlinx.cinterop.*
+
+@Suppress("MISSING_KDOC_TOP_LEVEL",
+    "MISSING_KDOC_CLASS_ELEMENTS",
+    "MISSING_KDOC_ON_FUNCTION"
+)
+actual class ProcessBuilder {
+    actual fun exec(command: List<String>, redirectTo: Path?): ExecutionResult {
+        val pd = popen!!.invoke(command.joinToString(" ").cstr.placeTo(MemScope()), "r".cstr.placeTo(MemScope()))
+
+        val stdout = buildString {
+            val buffer = ByteArray(4096)
+            while (true) {
```

Comment:
We can't write 
`while (fgets(buffer, buffer.size, pd)` instead of `while (true)`? 
buffer doesn't collect output information and fgets doesn't return exit code?

Reply:
Yep, I've forgotten how those C methods work :) Fixed.